src/signature.rs

```diff
@@ -70,10 +71,64 @@ impl Default for Signature {
     }
 }
 
+impl ToString for Signature {
+    // Constructs a string from a given signature
+    // The resulting string's length is 130
+    // first 32 bytes is "r" value
+    // second 32 bytes i s "s" value
+    // last byte is "v"
+    fn to_string(&self) -> String {
+        let r: [u8; 32] = self.r.clone().into();
+        let s: [u8; 32] = self.s.clone().into();
+        let mut wtr = vec![];
+        wtr.extend(&r);
+        wtr.extend(&s);
+
+        let v = self.v.to_bytes_be();
+        println!("vlen {}", v.len());
+        wtr.extend(&v[v.len() - 1..]);
+
+        let mut result = "0x".to_owned();
+        result += &bytes_to_hex_str(&wtr);
+        result
+        // format!("0x{:64x}{:64x}{:02x}", self.r, self.s, self.v)
```

Comment:
why not !trace?

Reply:
Oops. Actually it was my first attempt, but BigEndianInt misses some traits to make it work. I'll remove this one.